README.md

```diff
@@ -1,28 +1,25 @@
 <a rel="license" href=""><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
 
-# Coding for Reproducible Research Workshop Template
+# Intro to unix shell
 
-This repository is the template for creating websites for workshops that are part of the Coding for Reproducible Research initiative at UoE.  
+Please find the deployed site [here](https://uniexeterrse.github.io/intro-to-python/). These notes accompany in-person and online teaching during the two sessions of the Intro to Unix Shell course
 
-## How to create a workshop page
+## Source details
 
-To create a new website for a new workshop,  please use GitHub's "template" function to copy this `UoE-workshop-template` repository.
-**Please _do not fork this repository directly on GitHub._**
+This repository is based off of the University of Exeter `UoE-workshop-template` repository. This template uses Jekyll and GitHub pages to create a workshop guide.
 
-## Structure and required components
+## Ruby & local testing
 
-Each workshop has a landing or home page which provides the details of the workshop to individuals who might be interested in attending it. This home page will also direct participants who have signed up to set-up instructions and the course notes. It will also enable individuals who can't attend the workshop to all the resources and enable them to run through the material in their own time. 
+Jekyll is a ruby Gem. A simple Gemfile has been added. To test the site locally, clone the repository, and run the following (on macOS). Ensure `ruby` and `bundle` have been installed, and then run the following from the project root:
 
-Workshops are organised into 2-3 hour sessions of learning activity, where a workshop consists of 1-4 sessions. Beginners courses should have a minimum of 2 sessions.  Each session will be split into episodes, where episodes should last between 20-90 mins. Workshops and sessions will follow a pre-specified format. We recommend that there is an opportunity for a break approximately every 60-90 mins. 
+`bundle config set --local path 'vendor/bundle'`
```

Comment:
Add these three commands in a single fenced block:
```sh
bundle config set --local path 'vendor/bundle'
bundle install
bundle exec jekyll serve
```
or at least the last two (if the first is a MacOS thing)